### ISSUE TITLE
Watch loaded files & recompile (--watch options)

### ISSUE THREAD

**Issue by hybrist** [buggerjs/bugger#27]
When started with `--watch`, bugger should watch the files it loaded (source files for coffee-script) and recompile them when they change (hot code reloading).

**Comment by caspervonb** [comment]
:-1: please try to keep bugger as a pure v8/chrome protocol bridge only, this adds a lot of headaches and bulk.

Watching etc should be up to the clients to do.

**Comment by jkrems** [comment]
Yep, this is labelled "Icebox" for that reason. This was from before bugger worked correctly with Chrome workspaces (which mostly removes the need).
